adapter/internal/oasparser/model/constants.go

```diff
@@ -25,12 +25,15 @@ const (
 
 // Constants for OpenAPI vendor extension keys
 const (
-	productionEndpoints string = "x-wso2-production-endpoints"
-	sandboxEndpoints    string = "x-wso2-sandbox-endpoints"
-	xWso2BasePath       string = "x-wso2-basePath"
-	xWso2Label          string = "x-wso2-label"
-	xWso2Cors           string = "x-wso2-cors"
-	xThrottlingTier     string = "x-throttling-tier"
+	productionEndpoints  string = "x-wso2-production-endpoints"
+	sandboxEndpoints     string = "x-wso2-sandbox-endpoints"
+	xWso2BasePath        string = "x-wso2-basePath"
+	xWso2Label           string = "x-wso2-label"
+	xWso2Cors            string = "x-wso2-cors"
+	xThrottlingTier      string = "x-throttling-tier"
+	xAuthType            string = "x-auth-type"
+	xWso2DisableSecurity string = "x-wso2-disable-wso2"
```

Comment:
this extension seems wrong. Could we please check the mgw 3.2.0 docs and get the extension from there

Reply:
Fixes type https://github.com/wso2/product-microgateway/pull/1775/commits/0da4349b091d28e12161105106bf08fb3c279637